pkg/minikube/cluster/cluster.go

```diff
@@ -244,9 +244,8 @@ func createHost(api libmachine.API, config cfg.MachineConfig) (*host.Host, error
 	if err != nil {
 		if err == registry.ErrDriverNotFound {
 			exit.Usage("unsupported driver: %s", config.VMDriver)
-		} else {
-			exit.WithError("error getting driver", err)
 		}
+		exit.WithError("error getting driver", err)
```

Comment:
Is there anything more we can give the user here to understand what the driver issue might have been?

Reply:
I don't know yet -- I've never seen this particular call fail, so it's rather difficult to tell.